packages/taskarian/src/Task.ts

```diff
@@ -79,32 +79,80 @@ class Task<E, T> {
    *
    *     new Task([longFetchTask, timeoutTask])
    */
-  public static race<T, E>(ts: Array<Task<E, T>>): Task<E, T> {
-    if (ts.length === 0) {
-      return new Task((reject, resolve) => noop);
-    }
+  public static race<E, T>(tasks: ReadonlyArray<Task<E, T>>): Task<E, T> {
```

Comment:
Also fixed the type argument ordering here to conform to the order of the rest of the Task methods.